.gitignore

```diff
@@ -1,3 +1,6 @@
 custom/config.yml
 custom/people.opml
 custom/people.opml.bak
+
+/vendor/
+composer.lock
```

Comment:
Probably won't make any difference (considering that the exact version is specified in composer.json), but usually `composer.lock` is committed also, so that `composer install` installs the exact same versions as in development.